### DISCUSSION TITLE
Issues regarding drc count file

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#863] [General]
i'm trying to run drc for riscv32 design on asap7 pdk but tool couln't complete the run and didnt show any result on 6_drc.lyrdb and 6_drc_count.rpt files .I was using attached lydrc file .

**Comment by its-rakib** [comment]
![image](https://user-images.githubusercontent.com/127094304/224969127-1918f921-5162-4b6e-bd36-7cc26aff48b3.png)
[asap7.lydrc.txt](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/files/10967103/asap7.lydrc.txt)

**Comment by vijayank88** [comment]
DRC/LVS check with KLayout not enabled for ASAP7 pdk currently. check [this](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/discussions/854)
